static/search_events/style.css

```diff
@@ -0,0 +1,3 @@
+.width-90 {
```

Comment:
NIT In the future, you can choose a more specific name to this component, maybe more related to the table. For example, you can name this as `.table {...}` and put basic attributes, now only width: 90%, but then, you can more attributes if you want.